modules/android/pages/p2psync-multipeer.adoc

```diff
@@ -4,347 +4,656 @@ ifdef::show_edition[:page-edition: Enterprise Edition]
 :source-language: Java
 :source-language: Kotlin
 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// Ignore for now - latest changes in Swift branch
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-// 
-
-
-
-
-
-
-
-
-
-*Preview for this feature is on the iOS / Swift page.*
-
-////
 
 [abstract]
 {description}
 This approach requires minimal setup and automates peer discovery and connectivity management, making it simpler than xref:p2psync-websocket.adoc[active-passive P2P configurations].
 
 
+
+
 [#introduction]
 == Introduction
-// tag::introduction-full[]
-// tag::introduction[]
-Couchbase Lite's Peer-to-Peer synchronization solution offers secure storage and bidirectional data synchronization between mobile and IoT devices without needing a centralized cloud-based control point.
 
 
+Couchbase Lite's Peer-to-Peer synchronization solution offers secure storage and bidirectional data synchronization between mobile and IoT devices without needing a centralized cloud-based control point.
 
-For small mesh topologies, Multipeer P2P Replicator offers autodiscovery for wifi-based networks and secure communication via TLS and certificate-based authentication.
-The dynamic mesh topology gives optimal peer connectivity and the lightweight and low-maintenance configuration requires less management and less code than using Active-Passive peer-to-peer sync.
 
+For small mesh topologies, Multipeer P2P Replicator offers `autodiscovery` for wifi-based networks and secure communication via TLS and certificate-based authentication.
+The dynamic mesh topology gives optimal peer connectivity and the lightweight and low-maintenance configuration requires less management and less code than using Active-Passive peer-to-peer sync.
 
-// Bluetooth - not yet
 
 [TIP]
 .Bluetooth networking
 ====
-Multipeer Replication is currently available for use over IP networks.
+Multipeer Replication is available for use over IP networks.
 If you are working over a bluetooth network, then take a look at xref:p2psync-custom.adoc[integrating a Custom Listener]
 with Couchbase Lite's Active-Passive Peer-to-Peer synchronization.
 ====
 
 
+== Overview
+
+
+The Multipeer P2P Replicator automatically advertises its presence, discovers other peers,
+and establishes connections for data replication with peers that share the same group identifier defined by the application.
+
+
+To maintain optimal connectivity, efficient data transport, and balanced workloads, the Multipeer P2P Replicator forms a dynamic mesh network among peers in the same group.
+
+
+It avoids redundant direct connections, evenly distributes connections across peers, and optimizes communication paths.
+The mesh network continuously adapts as peers join or leave.
 
-// end::introduction[]
-// end::introduction-full[]
 
+// Mesh Diagram
+[mermaid]
```

Comment:
This won't work - you'll need the `graphviz` code from https://github.com/couchbase/docs-couchbase-lite/pull/981/files#diff-b0c06d07c6ec7535f7162c44682ea32e5b23e52da9cb587502f555a44a2fc4c4 (changed from iOS Device to Android).

Reply:
Updated to reflect this.